<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="1">
       <widget class="QSpinBox" name="cellCountInput">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="drawButton">
        <property name="text">
         <string>Rysuj</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Liczba kroków symulacji</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="simulationStepsInput">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Reguła</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="ruleInput">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Rozpocznij symulację</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Liczba komórek</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QTableView" name="celluralTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cellCountInput</tabstop>
  <tabstop>simulationStepsInput</tabstop>
  <tabstop>ruleInput</tabstop>
  <tabstop>startButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
